crates/hc_sandbox/src/ports.rs

```diff
@@ -56,11 +57,11 @@ pub(crate) async fn get_admin_api(
 async fn websocket_client_by_port(
     port: u16,
 ) -> std::io::Result<(WebsocketSender, tokio::task::JoinHandle<()>)> {
-    let (send, mut recv) = ws::connect(
-        Arc::new(WebsocketConfig::default()),
-        ([127, 0, 0, 1], port).into(),
-    )
-    .await?;
+    let req = holochain_websocket::ConnectRequest::new(([127, 0, 0, 1], port).into())
+        .try_set_header("Origin", "hc_sandbox")
```

Comment:
Allows the sandbox to be permitted when Holochain is deployed. It's a useful debugging tool but also perfectly reasonable to block it in production

Reply:
> the sandbox to be permitted when Holochain is deployed
What do you mean by "deployed" here? So the origin "hc_sandbox" *is* blocked in production (if not set otherwise in the conductor config)?

Reply:
I think hc_sandbox being blocked in production is desired as this would potentially allow orchestrators like the launcher come up with secret origins (that may change from session to session) to add an additional layer of security.

Reply:
I was thinking in the context of 'it's running on a holoport' here where administration should be done with Holo's tools and not `hc sandbox call` anyway. It's just an extra little bit of encouragement for things to be done intentionally rather than just using tools because they're present :)